network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/extensions/AbstractIidmExtension.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.network.store.iidm.impl.extensions;
+
+import com.powsybl.commons.extensions.AbstractExtension;
+import com.powsybl.iidm.network.Identifiable;
+import com.powsybl.iidm.network.Terminal;
+import com.powsybl.network.store.iidm.impl.Referrer;
+
+/**
+ * @author Geoffroy Jamgotchian {@literal <geoffroy.jamgotchian at rte-france.com>}
+ */
+public abstract class AbstractIidmExtension<I extends Identifiable<I>> extends AbstractExtension<I> implements Referrer<Terminal> {
```

Comment:
is it equvalent to AbstractMultiVariantIdentifiableExtension class in powsybl ?

Reply:
should we keep the same name ?

Reply:
in powsybl-core, AbstractMultiVariantIdentifiableExtension and AbstractIidmExtension are two different classes that exists together, and AbstractMultiVariantIdentifiableExtension just inherits from AbstractIidmExtension ...

In network-store, there is no AbstractMultiVariantIdentifiableExtension class ...
And keeping the same name for AbstractIidmExtension class just makes sense